webextension/background.js

```diff
@@ -82,10 +82,10 @@ function send_css(tab_id, iframe_css) {
     browser.tabs.sendMessage(tab_id,
 			     { generated_css: generated_css,
 			       user_css: prefs.useCustomCSS ?
-			                 prefs.customCSS : "" });
+			                 prefs.customCSS : "" }).catch(error => console.log("error with tab", tab_id, error));
```

Comment:
to avoid error in console.
console.log(...) can be commented or deleted

Reply:
Yeah, make the output commented out (this + the other 2 instances), so it serves as a reminder how to turn it on when debugging, but no reason to output if it can be silenced.